game-core/src/main/java/games/strategy/triplea/settings/BooleanClientSetting.java

```diff
@@ -30,6 +30,6 @@ public void saveSetting(final boolean value) {
 
   @Override
   public boolean getSetting() {
-    return getDefaultValue().orElse(false);
+    return getValue().or(this::getDefaultValue).orElse(false);
```

Comment:
Just noting that `ClientSetting#getValue()` already delegates to `getDefaultValue()` if required, so the `.or()` call here should be unnecessary.  Also, since this implementation always requires a default value to be specified (i.e. `defaultValue` is never `null`), the `.orElse(false)` will never be called.

In light of these two considerations, the implementation of this method should degenerate to just `getValueOrThrow()`.